contracts/hyphen/token/TokenManager.sol

```diff
@@ -172,6 +177,15 @@ contract TokenManager is ITokenManager, ERC2771ContextUpgradeable, OwnableUpgrad
         return transferConfig[tokenAddress];
     }
 
+    function setTokenSymbol(
```

Comment:
why chainid is needed here?? Why would we store token address and symbol mapping for other chain ids?

I am assuming we'd be just saving token address and symbol for current chainid only in the contract.